### PR TITLE
Docs: Update info on editors

### PR DESCRIPTION
**Pull request by microbit-mark**
Attempts to ease some of the confusion we have on support tickets where people are trying to open MicroPython projects in MakeCode.

Addresses comment in https://github.com/bbcmicrobit/micropython/pull/723#pullrequestreview-910674593 that this info is specific to all, not just V2 micro:bit devices

### CODE REVIEW

**Comment by microbit-carlos** [comment]
Preview URL: https://microbit-micropython--732.org.readthedocs.build/en/732/index.html

File: docs/index.rst [hunk 1]
```diff
@@ -16,20 +16,36 @@ This documentation includes lessons for teachers
...
+* `micro:bit Python Editor <https://python.microbit.org>`_ A simple
+  browser-based code editor, developed by the Micro:bit Educational
+  Foundation  and designed to help teachers and learners get the
+  most out of text-based programming on the micro:bit.
+
+* `Mu <https://codewith.mu>`_ A simple offline code editor for kids, teachers
+  and beginner programmers.
+
+* `uFlash <https://uflash.readthedocs.io/en/latest/>`_ A command line tool
+  for flashing raw Python scripts onto a BBC micro:bit.
```
Comment (microbit-carlos) [inline docs/index.rst:48]: We should keep a dash (`-`), or some other separator, between the link and description, no?

image

File: docs/tutorials/introduction.rst [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 
-Once Mu is installed connect your micro:bit to your computer via a USB lead.
+Write your script in the editor window and click the "Download" or "Flash" button to
+transfer it to the micro:bit.
 
-Write your script in the editor window and click the "Flash" button to transfer
-it to the micro:bit. If it doesn't work, make sure your micro:bit appears as
-a USB storage device in your file system explorer.
+If you have any problems with MicroPython or the editor, you can get support from the
+Micro:bit Educational Foundation team via `support.microbit.org <https://support.microbit.org>`_
```
Comment (microbit-carlos) [inline docs/tutorials/introduction.rst:14]: I think this sentence needs a full stop at the end:

image

Comment (microbit-mark) [reply docs/tutorials/introduction.rst]: updated in https://github.com/bbcmicrobit/micropython/pull/732/commits/51b8435df9ce9a08ac82982b012f453efd67f69a

File: docs/index.rst (hunk 1 above)
Comment (microbit-mark) [reply docs/index.rst]: updated in https://github.com/bbcmicrobit/micropython/pull/732/commits/51b8435df9ce9a08ac82982b012f453efd67f69a

File: docs/index.rst [hunk 3]
```diff
@@ -16,20 +16,36 @@ This documentation includes lessons for teachers
...
 you enjoy developing for the BBC micro:bit using MicroPython.
 
-If you're a new programmer, teacher or unsure where to start, begin with the tutorials.
+.. note::
 
-.. image:: comic.png
+    The BBC micro:bit MicroPython documentation contains information for all 
+    versions of the micro:bit board. Where functionality is applicable only
+    to the latest device, you will see a note or comment marking this as 
+    **V2**.
```
[the remark this answers is not in the document]
Comment (microbit-mark) [reply docs/index.rst]: removed in https://github.com/bbcmicrobit/micropython/pull/732/commits/51b8435df9ce9a08ac82982b012f453efd67f69a

[the remark this answers is not in the document]
Comment (microbit-mark) [reply docs/index.rst]: Note already exists in https://github.com/bbcmicrobit/micropython/blob/v2-docs/docs/index.rst

[the remark this answers is not in the document]
Comment (microbit-carlos) [reply docs/index.rst]: Great, thanks!

**Comment by microbit-carlos** [comment]
Thanks @microbit-mark! I think a VSCode config file was accidentally added to the PR, so I've pushed a commit to remove it, so this PR is good to go 🚀

**Pull request merged**